pkg/mssqldb/permissions.go

```diff
@@ -39,7 +40,7 @@ WHERE perms.state = 'G' OR perms.state = 'W'
 GROUP BY perms.grantee_principal_id, perms.state, principals.name, principals.type 
 ORDER BY perms.grantee_principal_id ASC 
 OFFSET @p1 ROWS FETCH NEXT @p2 ROWS ONLY`)
-
+	l.Debug("ListServerPermissions", zap.String("sql query: ", sb.String()))
```

Comment:
These can just be "sql query" and not "sql query: ". Zap formats these values appropriately.

Reply:
Fixed